### PR TITLE
BXMSDOC-8238 Propose a restructure of the Getting Started title

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: assemblies/assembly-getting-started-business-optimizer.adoc [hunk 1]
```diff
@@ -44,30 +44,7 @@ include::{enterprise-dir}/optimizer/optimizer-running-the-examples-in-an-IDE-pro
...
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-termination-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-using-rest-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-register-solver-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-call-solver-proc.adoc[leveloffset=+3]
-
-
+include::assembly-optaplanner-employee-rostering.adoc[]
 //Getting started with quarkus
 
 include::{enterprise-dir}/optimizer/optaplanner-quarkus-con.adoc[leveloffset=+1]
```
Comment (dupliaka) [inline assemblies/assembly-getting-started-business-optimizer.adoc:50]: I am a bit confused could you please point me out where this document could be found at
http://file.ork.redhat.com/~emmurphy/BXMSDOC-8238-pam

File: titles-enterprise/getting-started/title.adoc [hunk 2]
```diff
@@ -15,12 +15,13 @@ using a variety of available assets. You can also use {PLANNER} to find the opti
...
 
 // Assembly includes
+include::assemblies/assembly-introduction.adoc[]
 include::assemblies/assembly-getting-started-decision-services.adoc[]
 ifdef::PAM[]
 include::assemblies/assembly-getting-started-process-services.adoc[]
 include::assemblies/assembly-getting-started-case-management.adoc[]
 endif::[]
-include::assemblies/assembly-getting-started-business-optimizer.adoc[]
+include::assemblies/assembly-optaplanner-employee-rostering.adoc[]
```
Comment (dupliaka) [inline titles-enterprise/getting-started/title.adoc:24]: Generally I understand why do you change that, but I also see that in the documentation the paths are always point to optimizer like here: [assemblies/assembly-getting-started-business-optimizer.adoc](https://github.com/emmurphy1/kie-docs/pull/352/commits/d4aa0377bdd5a18aa43671af3ca15bfb9a835abe#diff-6b9e879c8928f609ba5a8cf93d86c2665c0593b2522c5cb03e1de79a89e8fb6b)

File: assemblies/assembly-getting-started-business-optimizer.adoc [hunk 3]
```diff
@@ -44,30 +44,7 @@ include::{enterprise-dir}/optimizer/optimizer-running-the-examples-in-an-IDE-pro
...
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-shift-required-skills-met-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-day-off-request-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-termination-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-using-rest-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-register-solver-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-call-solver-proc.adoc[leveloffset=+3]
-
-
+include::assembly-optaplanner-employee-rostering.adoc[]
```
Comment (dupliaka) [inline assemblies/assembly-getting-started-business-optimizer.adoc:47]: the same as https://github.com/emmurphy1/kie-docs/pull/352/commits/d4aa0377bdd5a18aa43671af3ca15bfb9a835abe#r833128758

File: assemblies/assembly-getting-started-business-optimizer.adoc (hunk 1 above)
Comment (emmurphy1) [reply assemblies/assembly-getting-started-business-optimizer.adoc:50]: @dupliaka here is the updated link with the anchor:
[V. Getting started with OptaPlanner in Business Central: An employee rostering tutorial](http://file.ork.redhat.com/~emmurphy/BXMSDOC-8325-pam/#assembly-optaplanner-employee-rostering_getting-started)

File: assemblies/assembly-getting-started-business-optimizer.adoc (hunk 3 above)
Comment (emmurphy1) [reply assemblies/assembly-getting-started-business-optimizer.adoc:47]: @dupliaka this is just for the PAM/DM Getting Started titles. The idea is to give users a walk-through tutorial of the different aspects of the product. Previously, the entire Getting Started section from the OP documentation was duplicated here. Now, I am just including the Employee Rostering tutorial in Business Central.

**Comment by emmurphy1** [comment]
This work has been moved to https://github.com/emmurphy1/kie-docs/pull/360

File: assemblies/assembly-getting-started-business-optimizer.adoc (hunk 3 above)
Comment (dupliaka) [reply assemblies/assembly-getting-started-business-optimizer.adoc:47]: ok

**Pull request closed without merging**